cypress/e2e/seededFlows/adminFlows/organizations/manageOrganizations.spec.js

```diff
@@ -0,0 +1,70 @@
+import { verifyAndDismissFlashMessage } from '../shared/adminUtilities';
+
+function openOrganizationOptions(callback) {
+  cy.findByRole('button', { name: 'Options' })
+    .should('have.attr', 'aria-haspopup', 'true')
+    .should('have.attr', 'aria-expanded', 'false')
+    .click()
+    .then(([button]) => {
+      expect(button.getAttribute('aria-expanded')).to.equal('true');
+      const dropdownId = button.getAttribute('aria-controls');
+
+      cy.get(`#${dropdownId}`).within(callback);
+    });
+}
+
+describe('Manage Organization Options', () => {
+  describe('As a super admin', () => {
+    beforeEach(() => {
+      cy.testSetup();
+      cy.fixture('users/adminUser.json').as('user');
+      cy.get('@user').then((user) => {
+        cy.loginAndVisit(user, '/admin/content_manager/organizations');
+      });
+
+      cy.findByRole('table').within(() => {
+        cy.findAllByRole('link', { name: '@Awesome Org' }).first().click();
+      });
+    });
+
+    it('should show a notice about the scheduled job', () => {
+      openOrganizationOptions(() => {
+        cy.findByRole('button', { name: 'Delete organization' }).click();
+      });
+
+      cy.getModal().within(() => {
+        cy.findByRole('button', { name: 'Delete organizion now' }).click();
+      });
+
+      verifyAndDismissFlashMessage(
+        `Your organization, "Awesome Org", deletion is scheduled.`,
+      );
+    });
+  });
+
+  describe('As an organization that has existing credits', () => {
```

Comment:
I can add another test that shows an error message when the user has only admin permissions and not super admin permissions, but we've called a user with `super_admin` permissions an `admin` in Cypress, and to update that is going to touch many many files so I'm going to try to see whats the best way to do this in another PR. However, the rails error message helper is being tested so it will cover a portion of the case of an admin user trying to delete the organization.

Reply:
Hmmm - adding a `singleResourceAdminUser` to represent non-superadmin admins might get the message across, but the helper spec also works